sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -81,14 +81,14 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit ed1f8bf618c88eaabea6bde0f6c06fc265f3b49f
-  # Merge: ca5c3421 69c43b45
+  set(UNIFIED_RUNTIME_REPO "https://github.com/Seanst98/unified-runtime.git")
```

Comment:
Just a reminder to bring back the correct repo and tag.

Reply:
Unified Runtime won't approve this PR until the tag has been updated to point towards the latest of UR. Only once it does will this PR gain approval from UR and merged in.